wiki_music/library/lyrics.py

```diff
@@ -94,7 +94,7 @@ def save_lyrics(tracks: List[str], types: List[str], band: str, album: str,
 
         if not lyr:
             for tr_k in tracks_dict.keys():
-                if fuzz.token_set_ratio(tr, tr_k) > 90:
+                if fuzz.token_set_ratio(tr, tr_k, score_cutoff=90):
```

Comment:
rapidfuzz has a score_cutoff parameter for all methods that returns 0 when the ratio is below the cutoff score